lib/main.dart

```diff
@@ -1,9 +1,30 @@
 import 'package:flutter/material.dart';
+import 'package:http/http.dart' as http;
+import 'dart:convert';
+import 'dart:developer' as developer;
 
 void main() {
   runApp(MyApp());
 }
 
+class CryptoCurrency {
```

Comment:
Should we extract this to another file?

Reply:
Yep, I will extract this code in the next commits.